### PR TITLE
MINOR: update stream docs for kip-134

### PR DESCRIPTION
**Pull request by dguy**
Add a section in the streams docs about the broker config introduced in KIP-134

### CODE REVIEW

**Comment by dguy** [comment]
@guozhangwang @miguno @bbejeck @mjsax.

File: docs/streams.html [hunk 1]
```diff
@@ -714,6 +715,24 @@
 settings.put(StremasConfig.producerConfig(ProducerConfig.RECEIVE_BUFFER_CONFIG), 64 * 1024);
 </pre>
 
+        <h4><a id="streams_broker_config" href="#streams_broker_config">Broker Configuration</a></h4>
+        <p>
+            Introduced in 0.11.0 is a new broker config that is particularly relevant to Kafka Streams applications, <code>group.initial.rebalance.delay.ms</code>.
+            This config specifies the time, in milliseconds, that the <code>GroupCoordinator</code> will delay the initial consumer rebalance.
+            The rebalance will be further delayed by the value of <code>group.initial.rebalance.delay.ms<</code> as each new member joins the consumer group, up to a maximum of the value set by <code>max.poll.interval.ms</code>.
```
Comment (bbejeck) [inline docs/streams.html]: typo extra '<'

**Comment by bbejeck** [comment]
just one small typo, otherwise LGTM

File: docs/streams.html [hunk 2]
```diff
@@ -714,6 +715,24 @@
...
+        <p>
+            In practice this means that if you are starting up your Kafka Streams app from a cold start, then when the first member joins the group there will be at least a 3 second delay before it is assigned any tasks.
+            If any other members join the group within the initial 3 seconds, then there will be a further 3 second delay.
+            Once no new members have joined the group within the 3 second delay, or <code>max.poll.interval.ms</code> is reached, then the group rebalance can complete and all current members will be assigned tasks.
+            The benefit of this approach, particularly for Kafka Streams applications, is that we can now delay the assignment and re-assignment of potentially expensive tasks as new members join.
+            So we can avoid the situation where one instance is assigned all tasks, begins restoring/processing, only to shortly after be rebalanced, and then have to start again with half of the tasks and so on.
+        </p>
+
+
+        <h4><a id="streams_execute" href="streams_execute">Executing Your Kafka Streams Application</a></h4>
```
Comment (miguno) [inline docs/streams.html]: I think this should read:
```
href="#streams_execute"
```
Note the `#`.

File: docs/streams.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
 
 <script><!--#include virtual="js/templateData.js" --></script>
 
-<script id="streams-template" type="text/x-handlebars-template">
+<!--<script id="streams-template" type="text/x-handlebars-template">-->
```
Comment (miguno) [inline docs/streams.html]: What's this?  This looks odd.

File: docs/streams.html [hunk 4]
```diff
@@ -926,7 +945,7 @@
             <li> JoinWindows has no default size anymore: JoinWindows.of("name").within(1000) changes to JoinWindows.of(1000) </li>
         </ul>
 
-</script>
+<!--</script>-->
```
Comment (miguno) [inline docs/streams.html]: What's this?  This looks odd.  Why is the `<script>` disabled now?

File: docs/streams.html (hunk 3 above)
Comment (dguy) [reply docs/streams.html]: oops!

File: docs/streams.html (hunk 4 above)
Comment (dguy) [reply docs/streams.html]: yeah - commented it out so i could test in browser. doh!

**Review by miguno — APPROVED** [review]

**Comment by dguy** [comment]
@guozhangwang

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and 0.11.0.

**Pull request closed without merging**